PSMDB-675 Documented restore conditions (#288)
percona/psmdb-docs: source/hot-backup.rst

.. rubric:: Restoring data from the backup

.. note::

>    To guarantee consistency of oplog data, restore the backup into the node with the replication enabled (the ``replication.replSetname`` option is specified in the configuration file) as oplog is available in replica sets and clusters only. If you need a standalone node, restart this node with the replication disabled after the restore. 

Reviewer: ```suggestion
   To guarantee consistency between the oplog and other collections you must restore the backup into a node with the replication enabled (i.e. the ``replication.replSetname`` option is specified in the configuration file).
```

I remove "Clusters" because they are besides the point. A hot backup is for one node's data only. One node can be a replica set by itself, but it can't be a cluster by itself.

I remove "if you need a standalone" part because if they want to do that, fine, they can just ignore the "local" db (it's only used for replication) and the oplog in it.
